app/components/support_interface/personal_details_component.rb

```diff
@@ -74,6 +76,8 @@ def right_to_work_or_study_row
       {
         key: 'Has the right to work or study in the UK?',
         value: RIGHT_TO_WORK_OR_STUDY_DISPLAY_VALUES.fetch(application_form.right_to_work_or_study),
+        action: 'right to work or study',
+        change_path: support_interface_application_form_edit_applicant_details_path(application_form),
```

Comment:
We shouldn't have change links to things that can't be edited yet.

Reply:
I think i was getting ahead of myself here 🤣